### PR TITLE
FIXED::Home page ui fix #9

### LINKED ISSUE #9: Home page ui fix
Fix the landing page of the home page and make it responsive in all screen sizes
If you wanna propose new home page style , make sure to attach the plans or the wireframe

### PR DESCRIPTION
**Pull request by nishant0708**
# Title
FIXED::Home page ui fix #9

- made page responsive using media queries 

#9

-  Bug fix (non-breaking change which fixes an issue)

---
# Checklist:
- [ y] I have checked to ensure there aren't other open Pull Requests for the same update/change?
- [ y] I have performed a self-review of my own code
- [y ] I have commented my code, particularly in hard-to-understand areas
- [y ] I have made corresponding changes needed to the documentation

### CODE REVIEW

**Comment by ArnabChatterjee20k** [comment]
@nishant0708 share images and screenshots

File: src/supabaseClient.js
Comment (ArnabChatterjee20k) [inline src/supabaseClient.js:1]: dont share environment keys and project details

**Comment by nishant0708** [comment]
https://github.com/ArnabChatterjee20k/PickPalette-PaletteFromImage/assets/101548649/9e7e79d0-8eb3-419a-af5c-f9fd5468ba42

**Comment by ArnabChatterjee20k** [comment]
Work on these points -
1) Add some more break points for large screens.
2) Work on the hero section. The TV part is not responsive(youtube player coming out)
image

**Comment by nishant0708** [comment]
hey i checked The TV part is already responsive for bigger screen . Can you please tell me more about the problem

**Comment by ArnabChatterjee20k** [comment]
> hey i checked The TV part is already responsive for bigger screen . Can you please tell me more about the problem

no the tv part is having issue in smaller screens

**Comment by nishant0708** [comment]
Please check this commit now
https://github.com/ArnabChatterjee20k/PickPalette-PaletteFromImage/assets/101548649/ac2e753d-eddd-4fa8-9dd9-3399e8ab8ae8

**Comment by ArnabChatterjee20k** [comment]
> Please check this commit now https://github.com/ArnabChatterjee20k/PickPalette-PaletteFromImage/assets/101548649/ac2e753d-eddd-4fa8-9dd9-3399e8ab8ae8

Can you please remove the supabase client file from the pr cause its making an unnecessary commit

**Comment by nishant0708** [comment]
Please Check Now

**Comment by nishant0708** [comment]
@ArnabChatterjee20k  please Check And suggest Changes

**Comment by ArnabChatterjee20k** [comment]
@nishant0708 can you provide another video showing the latest changes

**Comment by nishant0708** [comment]
for sure..

https://github.com/ArnabChatterjee20k/PickPalette-PaletteFromImage/assets/101548649/ed3a9a28-7365-480c-ade3-95ce4cbdd9c3

**Comment by ArnabChatterjee20k** [comment]
Great job
I am merging it

**Pull request merged**